AzureMonitorAgent/ama_tst/modules/high_cpu_mem/high_cpu_mem.py

```diff
@@ -0,0 +1,25 @@
+from error_codes          import *
+from errors               import is_error, print_errors
+from .check_logrot        import check_log_rotation
+from .check_usage         import check_usage
+
+def check_high_cpu_memory(interactive, prev_success=NO_ERROR):
+    print("CHECKING FOR HIGH CPU / MEMORY USAGE...")
+
+    success = prev_success
+
+    # check log rotation
+    print("Checking if log rotation is working correctly...")
+    checked_logrot = check_log_rotation()
+    if (is_error(checked_logrot)):
+        return print_errors(checked_logrot)
+    else:
+        success = print_errors(checked_logrot)
+
+    # check AMA CPU/memory usage
```

Comment:
Nitpick: I'd recommend printing a line like above saying "Checking AMA's CPU and memory usage..." or similar, so the user knows the check is occurring